requirements.txt

```diff
@@ -10,4 +10,4 @@ cryptography
 pyyaml
 pydantic
 python-magic
-nachet-datastore @git+https://github.com/ai-cfia/ailab-datastore.git@main
+nachet-datastore @git+https://github.com/ai-cfia/ailab-datastore.git@v1.0.0-nachet-datastore
```

Comment:
Just so I understand correctly, This point to a release on the Datastore related to Nachet?

Reply:
Exactly. it points to the following release : 
https://github.com/ai-cfia/ailab-datastore/releases/tag/v1.0.0-nachet-datastore
It only contains files dedicated to nachet. This PR is simply to fix currently workflow since the changes from datastore prevent current main branch from working. Once we validate that it works locally and we merge this, I will work on documenting the process.

Reply:
Thank you for the clarification! 🥇